ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -934,6 +957,10 @@ extension DrawerController: UIPopoverPresentationControllerDelegate {
     public func adaptivePresentationStyle(for controller: UIPresentationController, traitCollection: UITraitCollection) -> UIModalPresentationStyle {
         return .none
     }
+
+    public func popoverPresentationControllerShouldDismissPopover(_ popoverPresentationController: UIPopoverPresentationController) -> Bool {
+        return delegate?.shouldDismissDrawer?(self) != false
```

Comment:
I think if you do the following it'd be cleaner:

return delegate?.shouldDismissDrawer?(self) ?? true

Reply:
created `dismissPresentingViewController` to keep code cleaner.